libusb/os/windows_winusb.c

```diff
@@ -2607,6 +2595,25 @@ static int winusbx_configure_endpoints(int sub_api, struct libusb_device_handle
 	return LIBUSB_SUCCESS;
 }
 
+static HANDLE winusbx_ensure_dev_handle(struct libusb_device_handle *dev_handle, struct winusb_device_priv *priv, struct winusb_device_handle_priv *handle_priv, uint8_t iface)
+{
+	struct libusb_context* ctx = HANDLE_CTX(dev_handle);
+
+	if (handle_priv->interface_handle[iface].dev_handle == INVALID_HANDLE_VALUE) {
+		if ((priv->usb_interface[iface].path != NULL)
+			&& (priv->usb_interface[iface].apib->id == USB_API_WINUSBX)) {
```

Comment:
Can this be &&'d instead, since there are no /else/ clauses?

Reply:
I reduced nesting by inverting if. Looks cleaner that way

Reply:
Maybe it is worth using a helper variable for handle_priv->interface_handle[iface].dev_handle since it is repeated so much now?

Reply:
Would be nice but these are anonymous structs :/